src/main/scala/com/microsoft/hyperspace/index/IndexLogManager.scala

```diff
@@ -44,6 +44,9 @@ trait IndexLogManager {
   /** Returns the latest LogEntry whose state is STABLE */
   def getLatestStableLog(): Option[LogEntry]
 
+  /** Returns Active index log versions */
+  def getActiveIndexVersions(): Seq[Int]
```

Comment:
Does it make sense to pass `states: Seq[String]` to make it more generic? `def getIndexVersions(states: Seq[String]): Seq[Int]`?

Reply:
It's because I didn't expect any use case for other states; but will fix it.